Realm/RLMSyncSession.h

```diff
@@ -30,11 +32,59 @@ typedef NS_ENUM(NSUInteger, RLMSyncSessionState) {
     RLMSyncSessionStateInvalid
 };
 
+/**
+ The type of data transfer that a particular progress notification block will track.
+
+ Progress notification blocks can be registered on sessions if your app wishes to be informed
+ how many bytes have been uploaded or downloaded, for example to show progress indicator UIs.
+ */
+typedef NS_ENUM(NSUInteger, RLMSyncNotifierDirection) {
```

Comment:
The "sync notifier" terminology remained here. Perhaps `RLMSyncProgressDirection`?